mongock-cli-core/src/main/java/io/mongock/cli/core/commands/undo/UndoCommand.java

```diff
@@ -35,6 +35,9 @@ public Integer execution() throws Exception {
           mixinStandardHelpOptions = true,
           versionProvider = VersionProvider.class)
   public Integer all() {
+    if (!checkProfessionalBuilder()) {
+      return CommandLine.ExitCode.USAGE;
+    }
```

Comment:
Why is it done in here? Cannot we move it to the constructor in such a way that we can create a ProfessionalCommandBase